.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,36 @@
+name: GitHub Actions for Quantum Inspire Examples CI
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
```

Comment:
Again on this, sorry 😄! It seems that this should be triggering the workflow for this PR as is right now according to [the docs](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#example-using-multiple-events-with-activity-types-or-configuration).
There is also a question open from someone using the same snippet [here](https://github.com/actions/checkout/discussions/429). From [this](https://github.community/t/can-we-specify-the-default-branch-in-workflows/136258) I gather that this may be only working with templates.
Shall we try something like [this](https://stackoverflow.com/a/68414395/893239)?

Reply:
Looks good, I will change